### DISCUSSION TITLE
Added href to FAB buttons

### DISCUSSION THREAD

**Discussion by treeder** [treeder/material#15] [Announcements]
The original one didn't have href attribute for a FAB for some reason and didn't plan on it. https://github.com/material-components/material-web/discussions/5358

This is now part of this library:

```html
  <md-fab variant="primary" href="/abc">
    <md-icon slot="icon">add</md-icon>
  </md-fab>
```

**Comment by Shamimwn** [comment]
How can you don't join the greatest news website how dare you 
Https://www.shamimwn.ir
